app/src/js/components/AsyncCommands/AsyncCommands.js

```diff
@@ -104,33 +105,32 @@ class AsyncCommand extends React.Component {
           'modal__container--onscreen': confirmModal
         })}>
           <DefaultModal
-            className='async-modal'
+            className='async__modal--confirm'
             onCancel={this.cancel}
             onCloseModal={this.cancel}
             onConfirm={this.confirm}
             title={text}
-            children={(
-              <div className='modal__internal modal__formcenter'>
-                { confirmOptions ? (confirmOptions).map(option =>
-                  <div key={`option-${confirmOptions.indexOf(option)}`}>
-                    {option}
-                    <br />
-                  </div>
-                ) : null }
-                <h4>{confirmText}</h4>
-              </div>
-            )}
             showModal={confirmModal}
-          />
+          >
+            <div className='modal__internal modal__formcenter'>
+              { confirmOptions && (confirmOptions).map(option =>
+                <div key={`option-${confirmOptions.indexOf(option)}`}>
+                  {option}
+                  <br />
+                </div>
+              )}
+              <h4>{confirmText}</h4>
+            </div>
+          </DefaultModal>
           <DefaultModal
-            className='link--no-underline'
-            onCancel={this.cancel}
+            className='async__modal--success'
+            onCancel={success}
```

Comment:
What does success mean now? It didn't make sense to me that this is connected to `onCancel` now

Reply:
Yeah this is confusing. The modal has two buttons, cancel and confirm. You can use one or the other. Changed this to use the confirm button so that we're using `onConfirm` instead of `onCancel` for clarity.